### PR TITLE
Spanish Translate [WIP]

### PR DESCRIPTION
**Pull request by Cadiducho**
Hi, I'm working on translate this _awesome_ plugin to Spanish, my native language.
So, I think in one or two days will be finished, but... May plugin won't work with UTF-8 chars? Like 'ñ', 'ó' and similar used in Spanish.... I hope everything will work correctly

### CODE REVIEW

File: src/main/java/bammerbom/ultimatecore/bukkit/commands/CmdJail.java [hunk 1]
```diff
@@ -116,7 +116,11 @@ public void run(final CommandSender cs, String label, String[] args) {
             Location loc = UC.getServer().getJail(jail);
             if (pl.isOnline()) {
                 LocationUtil.teleportUnsafe(pl.getPlayer(), loc, TeleportCause.PLUGIN);
-                r.sendMes(pl.getPlayer(), "jailTarget", "%Time", time != -1L ? DateUtil.format(time + System.currentTimeMillis()) : r.mes("jailEver"));
```
Comment (Cadiducho) [inline src/main/java/bammerbom/ultimatecore/bukkit/commands/CmdJail.java]: I had to change this line to a better interpretation of the message in Spanish. I hope it's not a problem

File: src/main/resources/Messages/EN.properties [hunk 2]
```diff
@@ -170,7 +170,7 @@ jailAlreadyJailed=@2%Player @3is already jailed!
 jailNotFound=@3Jail not found: @2%Jail
 jailSender=Jailed @2%Player @1in jail @2%Jail @1for @2%Time
 jailTarget=@1You have been jailed for @2%Time@1.
-jailEver=@2ever
+jailTargetEver=@1You have been jailed @1forever@1.
```
Comment (JonathanBrouwer) [inline src/main/resources/Messages/EN.properties]: Remove the unneeded @1 tags.

**Comment by JonathanBrouwer** [comment]
BTW, my english is not that good, so if you find any bad english you can fix it.

**Comment by JonathanBrouwer** [comment]
Sorry for fucking up your pull request with the latest commit.

**Comment by Cadiducho** [comment]
No problem, that is now fixed :+1:

**Comment by JonathanBrouwer** [comment]
The lowerCamelCase is not that much work, it are just these six upper keys that changed:
PlayerNotFound, WorldNotFound, NumberFormat, NotPlayer, NoPermissions, UnknownCommand.

The sorting is easy with http://alphabetizer.flap.tv/ or another online tool.

**Comment by Cadiducho** [comment]
sorry for the absence, I've been busy with exams. the translation is still active

**Comment by JonathanBrouwer** [comment]
No problem.

File: src/main/resources/Messages/ES.properties [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+uptimeMessage=@1Uptime: @2%Time
+uuidUsage=@3/usage @2<Player>
+uuidMessage=@1The uuid of @2%Name @1is @2%Uuid@1.
+vanishSelf=@1Vanish mode is now @2%Status@1.
+vanishSelfT=@1Vanish mode is now @2%Status @1for @2%Time@1.
+vanishOthersSelfMessage=@1Your vanish mode from @2%Player @1is now @2%Status@1.
+vanishOthersOtherMessage=@1Your vanish mode is now @2%Status@1.
+vanishOthersSelfMessageT=@1Your vanish mode from @2%Player @1is now @2%Status @1for @2%Time@1.
+vanishOthersOtherMessageT=@1Your vanish mode is now @2%Status @1for @2%Time@1.
+vanishTarget=@1You are now vanished.# To change this license header, choose License Headers in Project Properties.
```
Comment (JonathanBrouwer) [inline src/main/resources/Messages/ES.properties:475]: Weird note?

**Comment by JonathanBrouwer** [comment]
Closed because of inactivity.
If you wish to continue comment on this issue.

**Pull request closed without merging**